google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -48,8 +48,8 @@ appengine.cloudsdk.location.label=Cloud SDK directory\:
 appengine.cloudsdk.location.browse.window.title=Browse to Cloud SDK directory
 appengine.cloudsdk.location.badchars.message=The specified Cloud SDK directory contains invalid characters or starts or ends with spaces.
 appengine.cloudsdk.location.invalid.message=No Cloud SDK was found in the specified directory.
+appengine.cloudsdk.location.invalid.message.deploy=The Cloud SDK is misconfigured. To fix, reconfigure the Server above.
```

Comment:
oh, I missed that. I was just thinking that since the message is already context sensitive, why not have the message name also indicate the context.